samples/word/01-basics/search.yaml

```diff
@@ -25,39 +25,36 @@ script:
                         return context.sync();
                     })
             })
-                .catch(function(e){
-                    console.log(e.message);
-                })
+                .catch(OfficeHelpers.Utilities.log);
         }
 
         function wildcardSearch() {
             Word.run(function (context) {
                 // check out how wildcard expression are built, also use the second parameter of the search method to include search modes (i.e. we are using wildcards).
-                var results = context.document.body.search("$*.[0-9][0-9]", {matchWildcards:true} );
+                var results = context.document.body.search("$*.[0-9][0-9]", { matchWildcards: true });
                 context.load(results);
                 return context.sync()
                     .then(function () {
                         //lets traverse the search results... and highlight..
                         for (var i = 0; i < results.items.length; i++) {
                             results.items[i].font.highlightColor = "red";
-                             results.items[i].font.color = "white";
+                            results.items[i].font.color = "white";
                         }
                         return context.sync();
                     })
             })
-                .catch(function(e){
-                    console.log(e.message);
-                })
+                .catch(OfficeHelpers.Utilities.log);
         }
 
-        function setup() { 
+        function setup() {
             Word.run(function (context) {
                 // lets insert a couple of paragraphs to illustrate the point..
                 context.document.body.clear();
-                context.document.body.insertParagraph("Video provides a powerful way to help you prove  your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.", "start"); 
+                context.document.body.insertParagraph("Video provides a powerful way to help you prove your point. When you click Online Video ($10,000.00), you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.", "start");
```

Comment:
no i needed to add a currency formatted text on the sample text, so the search by wildcard example can find it,.